src/Streamly/Internal/Data/Stream/Parallel.hs

```diff
@@ -386,6 +395,22 @@ tapAsync f m = mkStream $ \st yld sng stp -> do
     sv <- newFoldSVar st f
     foldStreamShared st yld sng stp (teeToSVar sv m)
 
+-- | Concurrently distribute a stream to a collection of fold functions,
+-- discarding the outputs of the folds.
+--
+-- >>> S.drain $ distributeAsync_ [S.mapM_ print, S.mapM_ print] (S.enumerateFromTo 1 2)
```

Comment:
We can have a better example maybe with a delay in the `Fold` or different folds in the container